lib/prompts.js

```diff
@@ -55,6 +54,15 @@ var prompts = [
     choices: function(answers) {
       return answers['drupalDistro'].versions;
     }
+  },
+  {
+    type: 'confirm',
+    name: 'useComposerManager',
+    message: 'Use Composer Manager to manage composer dependencies?',
+    default: false,
```

Comment:
I think we should default to on. Most projects at one point or another are going to need to include either a custom or contrib module that utilizes a `composer.json` file (eg, `search_api_solr`, `chosen`, etc), and adding this functionality in after-the-fact seems like more work than removing it later.